configs/projects/_shared-pe-bolt-server.rb

```diff
@@ -148,6 +148,7 @@
 proj.component('rubygem-ruby_smb')
 proj.component('rubygem-rubyzip')
 proj.component('rubygem-scanf')
+proj.component('rubygem-sys-filesystem')
```

Comment:
We dont ship mount_core in pe so i dont think we need to include for bolt-server. https://github.com/puppetlabs/pe-tasks-vanagon/blob/2019.8.x/configs/projects/pe-tasks.rb

Reply:
Removed for bolt-server